pkg/utils/version/vars_test.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"testing"
+
+	"sigs.k8s.io/kwok/pkg/consts"
+)
+
+func TestVersionInfo(t *testing.T) {
+	testCases := []struct {
+		input  string
+		output string
+	}{
+		{"v1.2.3", "1.2.3-alpha"},
+		{"1.2.3", "1.2.3-alpha"},
+		{"v0.1.0", "0.1.0-alpha"},
+		{"v0.1.0-alpha", "0.1.0-alpha"},
+	}
+
```

Comment:
```suggestion
originVersion := consts.Version
defer func() {
  consts.Version = originVersion
} ()
```

Return to original state after testing to avoid disturbing other tests

Reply:
Or modify the `versionInfo` to `versionInfo(version string, preRelease string)`

Reply:
done , I have some change to this unit test ,please review again